app/controllers/bookmarks.js

```diff
@@ -6,6 +7,25 @@ import { Promise } from 'rsvp';
 export default Controller.extend({
   mainMap: service(),
   metrics: service(),
+  router: service(),
+
+  savedLayerSets: window.localStorage['saved-layer-sets']
```

Comment:
This might be more trouble than it's worth but did you consider making an Ember model for these bookmarkable layer sets, similar to how we have a `bookmark` model already? This approach isn't bad but using a model might be more Ember-y

Reply:
I considered it, but abandoned the idea after looking at the way the bookmarks were set up and finding the abstraction was a little much for me.

Reply:
@dhochbaum-dcp Have you spent any more time on the approach I described here? If you have and are still hitting a wall, I won't hold up the PR over it.

Reply:
Ok. I'll take another look at the PR as is and leave a review.